presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -523,7 +525,17 @@ public SystemSessionProperties(
                         QUERY_MAX_TOTAL_MEMORY_PER_NODE,
                         "Maximum amount of total memory a query can use per node",
                         nodeMemoryConfig.getMaxQueryTotalMemoryPerNode(),
-                        true));
+                        true),
+                integerProperty(
+                        DYNAMIC_FILTERING_MAX_PER_DRIVER_VALUE_COUNT,
+                        "Experimental: maximum number of build-side rows to be collected for dynamic filtering per-driver",
+                        featuresConfig.getDynamicFilteringMaxPerDriverRowCount(),
+                        false),
+                dataSizeProperty(
+                        DYNAMIC_FILTERING_MAX_PER_DRIVER_BYTE_SIZE,
+                        "Experimental: maximum number of bytes to be collected for dynamic filtering per-driver",
```

Comment:
Would be better to drop `bytes` from here as well

Reply:
Thanks, fixed.